build/images/Dockerfile.base.ubuntu

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:18.04 as cni-binaries
```

Comment:
I feel like this Dockerfile should go to the `build/images/base` directory

Reply:
Moved.